test/e2e_node/cpu_manager_test.go

```diff
@@ -710,6 +774,21 @@ func runCPUManagerTests(f *framework.Framework) {
 		runSMTAlignmentPositiveTests(ctx, f, smtLevel)
 	})
 
+	ginkgo.It("should not enforce CFS quota for containers with static CPUs assigned", func(ctx context.Context) {
+
+		newCfg := configureCPUManagerInKubelet(oldCfg,
+			&cpuManagerKubeletArguments{
+				policyName:                       string(cpumanager.PolicyStatic),
+				reservedSystemCPUs:               cpuset.New(0),
+				enableCPUManagerOptions:          true,
```

Comment:
should not be related to the test failure, but you probably don't need to enable the options here

Reply:
removed!